axiom/test/test_axiomatic.py

```diff
@@ -317,6 +328,76 @@ def test_reactorSelection(self):
         return complete
 
 
+    def test_reactorSelectionLongOptionStyle(self):
+        """
+        L{AxiomaticStart} optionally takes the name of a reactor in the form
+        --reactor=[shortName] and installs it instead of the default reactor.
+        """
+
+        axiomatic = self._getAxiomaticScript()
+
+        # Create a store for the child process to use and put an item in it.
+        # This will force an import of the module that defines that item's
+        # class when the child process starts.  The module imports the default
+        # reactor at the top-level, making this the worst-case for the reactor
+        # selection code.
+        storePath = self.mktemp()
+        store = Store(storePath)
+        SomeItem(store=store)
+        store.close()
+
+        # Install select reactor because it available on all platforms, and
```

Comment:
Missing word: "it ~~is~~ available". I see this isn't your error since it was copied from the other test, but I think we might as well fix it now (and in the other comment).